content/javascript/concepts/dates/terms/setUTCMilliseconds/setUTCMilliseconds.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
The spacing of this entry is incorrect. There should never be more than one empty line between content. Please review one of our templates or use another existing entry as a reference.
https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md